scripts/generator/graphql_generator/csharp/Cart.cs.erb

```diff
@@ -102,6 +103,52 @@ namespace <%= namespace %> {
             });
         }
 
+        public void SetShippingLine(string shippingRateHandle, Action<ShopifyError> callback) {
```

Comment:
SDK user can use them if they make their own Graph queries, and extend Cart. I'll make the change to delegate